portfolio/src/main/webapp/script.js

```diff
@@ -144,6 +144,7 @@ function createListElement(comment, nickname) {
   let h3Element = document.createElement('h3');
   let h5Element = document.createElement('h5');
   let pElement = document.createElement('p');
+  let iElement = document.createElement('i');
```

Comment:
I'd avoid `<i>` and maybe use `<span>` + css instead.  In case if you change your mind later on, you'd then only have to change your css.  Instead of having to change html+javascript+css.